dts/arm/ti/mspm0g3xxx.dtsi

```diff
@@ -66,7 +66,7 @@
                 timg0: timg0@40084000 {
                     compatible = "ti,mspm0-counter";
                     status = "disabled";
-                    clock-frequency = <32000000>;
+                    clocks = <&sysclock>;
```

Comment:
not sysclk?

Reply:
you are right. typo while doing the changes..